deployment_scripts/linode-marketplace-mysql/test-vars.sh

```diff
@@ -26,6 +26,12 @@ else
         UDF_VARS["DATABASE"]="mysql" # default
 fi
 
+if [[ -n "${ADD_ONS}" ]]; then
+        UDF_VARS["ADD_ONS"]="${ADD_ONS}"
+else
+        UDF_VARS["ADD_ONS"]="none" # default
```

Comment:
```suggestion
        UDF_VARS["ADD_ONS"]="[${ADD_ONS}]"
else
        UDF_VARS["ADD_ONS"]="['none']" # default
```
gotta double check this